vitess.io/css/site.scss

```diff
@@ -341,45 +541,21 @@ li {
 }
 #centerTextCol {
   h2, h3, h4, h5, h6 {
-    // We need to set padding-top:50px so heading text isn't hidden
-    // by the nav bar when linking to a specific anchor. We then set
-    // a negative margin-top to try to close some of the gap back up,
-    // but we can only go as far as the margin-bottom of the previous
-    // paragraph, or else it will overlap and prevent links above from
-    // being clicked.
     padding-top: 50px;
-    margin-top: -14px;
+    margin-top: -50px;
```

Comment:
I looked at your preview instance, and this seems to have reintroduced the problem I mentioned in the comment. For example, on concepts.html, at the end of the **Keyspace** section, there's a link on the text "sharded". That link can't be clicked, unless you hover over the very top portion of it, because the bounding box for the **Keyspace id** header below overlaps and steals the click.